### PR TITLE
[WIP] Remove modification tree visibility indicator

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

**Comment by Meklo** [comment]
The weird amount of commit is, I think, due to the fact I based my local branch on top of other branches which weren't merged when I started working on this subject

File: src/translations/fr.json [hunk 1]
```diff
@@ -286,6 +286,7 @@
     "NodeUpdateError": "Une erreur est survenue lors de la mise à jour du nœud",
     "NodeBuildingError": "Une erreur est survenue lors de la construction du nœud",
     "NetworkModifications": "Modifications de réseau",
+    "NodeParentRetrievalError": "La récupération du noeud visualisable le plus proche a échouée",
```
Comment (etiennehomer) [inline src/translations/fr.json:289]: -e

**Comment by etiennehomer** [comment]
Warning: the incrementation of the nodes' name is broken. When we create a new node, its name should be "New node 0" but we have just "New node" now. So that we have to change the node name before creating a new one.

**Comment by Meklo** [comment]
> Warning: the incrementation of the nodes' name is broken. When we create a new node, its name should be "New node 0" but we have just "New node" now. So that we have to change the node name before creating a new one.

My branch wasn't up to date with the main one. 
Merging it fixed this issue

File: src/components/graph/nodes/network-modification-node.js [hunk 2]
```diff
@@ -119,9 +116,13 @@ const NetworkModificationNode = (props) => {
                                 style={{ margin: 'auto' }}
                             />
                         )) ||
-                            (props.data.buildStatus === 'NOT_BUILT' && (
-                                <VisibilityOffIcon style={{ margin: 'auto' }} />
-                            ))}
+                            (props.data.buildStatus === 'NOT_BUILT' &&
+                                //Setup for a different display mode, replace false by a configuration variable
+                                false && (
```
Comment (etiennehomer) [inline src/components/graph/nodes/network-modification-node.js:121]: I think you can remove it if this is not used for the moment.
It is planned in short term to implement another display mode ?

File: src/components/graph/nodes/network-modification-node.js [hunk 3]
```diff
@@ -142,7 +143,11 @@ const NetworkModificationNode = (props) => {
 
             <div className={classes.outOfBoundIcons}>
                 {props.data.readOnly && <LockIcon />}
-                {props.id === workingNode?.id && <VisibilityIcon />}
+
+                {
+                    //Setup for a different display mode, replace false by a configuration variable
+                    false && props.id === workingNode?.id && <VisibilityIcon />
```
Comment (etiennehomer) [inline src/components/graph/nodes/network-modification-node.js:149]: Same

**Comment by etiennehomer** [comment]
> > Warning: the incrementation of the nodes' name is broken. When we create a new node, its name should be "New node 0" but we have just "New node" now. So that we have to change the node name before creating a new one.
> 
> My branch wasn't up to date with the main one. Merging it fixed this issue

Ah yes, it works fine

File: src/components/graph/nodes/network-modification-node.js (hunk 2 above)
Comment (etiennehomer) [reply src/components/graph/nodes/network-modification-node.js:121]: Ok I just had the answer in the backlog refinement :) So you can keep it yes

**Pull request closed without merging**